decoders/connector/daytec/siglog_32a2/v1.0.0/payload.test.ts

```diff
@@ -0,0 +1,16 @@
+import { describe, test, expect } from "vitest";
```

Comment:
Please correct the unit test and ensure it passes by running npm run test in your terminal.